spec/Section 4 -- Introspection.md

```diff
@@ -146,6 +146,9 @@ type __Type {
 
   # should be non-null for NON_NULL and LIST only, must be null for the others
   ofType: __Type
+
+  # should be non-null for SCALAR only, must be null for the others
```

Comment:
Nitpick: Currently in my draft for the reference implementation https://github.com/graphql/graphql-js/pull/2276, if you don't add the `@specified` directive to a scalar, `specifiedBy` will be `null`.

If that's the correct implementation, then perhaps this wording could be updated to something along the lines of:
```suggestion
  # should be a string or null for SCALAR only, must be null for the others
```

Reply:
Hmm, it's a good point but I think I'd prefer just replacing `should` with `can`.